omnibus/Gemfile

```diff
@@ -20,3 +20,7 @@ group :development do
   gem "kitchen-vagrant", ">= 1.3.1"
   gem "winrm-fs", "~> 1.0"
 end
+
+if RUBY_PLATFORM.match?(/mswin|mingw|windows/)
+  gem win32-process
```

Comment:
@johnmccrae I think that needs quotes?

Reply:
You are correct, sir. It did need them. Ha